packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -249,6 +256,7 @@ export function PanelChrome({
     <section
       className={cx(styles.container, { [styles.transparentContainer]: isPanelTransparent })}
       style={containerStyles}
+      aria-labelledby={panelTitleId}
```

Comment:
@ckbedwell This is allowed for `landmark` roles, so have no idea why pa11y is complaining 🤔 Any thoughts?

Reply:
Ah, makes sense 🤦 Thank you for clarifying. "No element with that id" would've been a better error message than "It's invalid"

Reply:
Tell me about it 😂  It took me a while to work out what the problem was.